.github/workflows/ci.yml

```diff
@@ -47,3 +47,6 @@ jobs:
 
       - name: Build with Gradle
         run: ./gradlew check --parallel
+
+      - name: API check
```

Comment:
Correct me if I'm wrong but I think `check` already depends on `apiCheck`, so there would be no need to call it explicitely.

Reply:
It does!